core/kernel/persistence/smoothsql/class.SmoothRdf.php

```diff
@@ -105,4 +157,34 @@ protected function getModel()
     {
         return $this->model;
     }
+
+    /**
+     * @param core_kernel_classes_Triple $triple
+     */
+    private function checkResourceTriggers(core_kernel_classes_Triple $triple)
```

Comment:
```suggestion
    private function triggerResourceCreatedEvent(core_kernel_classes_Triple $triple)
```

Reply:
as we discussed `watchResourceCreated` will be there